client/lib/feed-stream-store/index.js

```diff
@@ -45,6 +45,11 @@ function mixedKeyMaker( post ) {
 	return siteKeyMaker( post );
 }
 
+const recommendedKeyMaker = post => ( {
+	...siteKeyMaker( post ),
+	isRecommendation: true }
```

Comment:
We had an issue recently where adding fields to `postKey` broke individual post fetches.  I think because we were adding the entire postKey object to the querystring.

The TODO is to remind me to check if its still doing that / It breaks things.